test/libsolidity/smtCheckerTests/modifiers/modifier_code_after_placeholder.sol

```diff
@@ -8,8 +8,6 @@ contract C
 		require(x > 0);
 		_;
 		// Fails because of overflow behavior.
-		// Overflow is not reported because this assertion prevents
```

Comment:
Does this mean that this feature was removed?

Reply:
Nice catch. At some point I had removed adding the assertion as a constraint before verifying which reports more assertion fails in some cases but then removes this feature. I'll put it back.